.github/workflows/publishing.yml

```diff
@@ -836,6 +836,55 @@ jobs:
             exit $status_sum
           fi
   
+  wheel_validation_x86_simple:
+    name: Simple Python wheel installation validation
+    needs: [assets, cudaq_wheels]
+
+    runs-on: linux-amd64-gpu-v100-latest-1
+
+    container:
+      image: ubuntu:22.04
+      options: --user root
+      env:
+        NVIDIA_VISIBLE_DEVICES: ${{ env.NVIDIA_VISIBLE_DEVICES }}
+        TERM: xterm
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.github_commit || needs.assets.outputs.github_commit }}
+
+      - name: Load wheel
+        uses: actions/download-artifact@v4
+        with:
+          name: x86_64-py3.10-wheels
+          path: /tmp/install
+      
+      - name: Run simple x86 validation
+        shell: bash
+        run: |
+          # These simple steps are only expected to work for x86 and only for
+          # targets and test cases that don't require MPI.
+          # Create clean python3 environment.
+          apt-get update && apt-get install -y --no-install-recommends python3 python3-pip
+          pip install /tmp/install/cuda_quantum-*.whl
+          status_sum=0
+          set +e # Allow script to keep going through errors
+          # Verify that the necessary GPU targets are installed and usable
+          for tgt in nvidia nvidia-fp64 tensornet; do
```

Comment:
I think it would be good to run at least a simple program here. Otherwise some dependencies might not be correctly picked up but go unnoticed since they are not needed/loaded when merely setting the target.

Reply:
Updated in daab30b13